iothub/device/src/InternalClient.cs

```diff
@@ -183,6 +185,12 @@ public int DiagnosticSamplingPercentage
         // Codes_SRS_DEVICECLIENT_28_002: [This property shall be defaulted to 240000 (4 minutes).]
         public uint OperationTimeoutInMilliseconds { get; set; } = DeviceClient.DefaultOperationTimeoutInMilliseconds;
 
+        internal string PnPModelId
+        {
+            get => _pnpOptions.ModelId;
```

Comment:
can _pnpOptions be null ?

Reply:
not as per current design. PnPOptions is currently an internal type, which is always initialized.